modules/cert-manager-fips-procedure.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/index.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cert-manager-fips-procedure_{context}"]
+= Enabling FIPS Compliance for {cert-manager-operator} on {product-title}
+
+To enable FIPS compliance for the {cert-manager-operator} on {product-title} using the `check-payload` tool, follow the procedure.
+
+.Procedure
+
+. Review the internal guidance documents on FIPS compliance. See "OpenShift security guide book".
+
+. Update the Cluster Service Version (CSV) file with the necessary annotations to trigger FIPS compliance checks.
+
+. Clone and build the `check-payload` tool using link:https://github.com/openshift/check-payload[check-payload tool].
+
+[source,terminal]
+----
+$ git clone https://github.com/openshift/check-payload.git
+$ cd check-payload
+$ make
+----
+
+. Use the `check-payload` tool to scan container images and ensure they meet FIPS requirements using the following command:
+
+[source,terminal]
+----
+$ sudo ./check-payload scan payload -V <OpenShift version> --url <payload URL> --output-file report.txt
+----
+
+. Ensure the following during the scan:
+
+* Enable the use of cgo using `CGO_ENABLED=1`.
+* Ensure OpenSSL library is included.
+* Enforce strict FIPS mode at runtime.
+* Ensure no conflicting `OpenSSL` version.
+* Verify binaries are dynamically linked.
+
+. Add `check-payload` tool to your CI workflows to automate FIPS compliance checks.
+
+. Configure your CI environment to run with FIPS enabled.
```

Comment:
As discussed, we can consider removing this section about the check-payload tool. Essentially, that'd strip out the Procedure section IMO.

Reply:
Yes. as discussed, I will remove the section.